sales-forecasting/wave_forecast.py

```diff
@@ -211,15 +194,23 @@ async def initialize_app(q: Q):
 
     plot_data = q.app.sales_data.get_plot_data(**asdict(q.app.user_inputs))
 
-    del q.page['loading']
     q.page['sidebar'] = ui.form_card(
         box='sidebar',
         items=get_user_input_items(q.app.sales_data, q.app.user_inputs)
     )
     await draw_weekly_sales_plot(q, plot_data)
 
+def on_startup():
```

Comment:
@mturoci Could you please test this on cloud.h2o.ai, I have had some issues with `on_startup` and deployment in the past and just want to double check that this works as intended. Thank you!

Reply:
Sure, thanks for pointing this out.

Reply:
Verified, working as expected.

Reply:
Thanks!